css/Comments.css

```diff
@@ -0,0 +1,3 @@
+.bulkmanagerheading .title {
```

Comment:
I'm not sure that the ajax loading mechanism unloads css files once you navigate away from this screen.